### PR TITLE
Googlenet

### PR DESCRIPTION
**Pull request by akeeei**
1、修改main.py增加对output是否为tuple的校验
2、googlenet模型迁移

### CODE REVIEW

File: cv/classification/main.py
Comment (ShawnXuan) [inline cv/classification/main.py:1]: 这个文件不用修改

Comment (akeeei) [reply cv/classification/main.py:1]: 已改回目前仓库内版本，不过用pre-commit修正了下格式

Comment (ShawnXuan) [reply cv/classification/main.py:1]: 还是保持以前的状态。
一个pr只解决一个问题，pr修改的文件只限定到这个问题覆盖的范围，这样多人协作一个项目时就减少了冲突。

**Comment by akeeei** [comment]
image

**Comment by akeeei** [comment]
infer更正，推理结果：
INFO  * Acc@1 70.132 Acc@5 89.728
INFO Accuracy of the network on the 1563 test images: 70.1%
INFO throughput averaged with 30 times
INFO batch_size 32 throughput 1684.5348415445303

File: cv/classification/config.py [hunk 1]
```diff
@@ -34,7 +34,8 @@
 _C.DATA.PIN_MEMORY = True
 # Number of data loading threads
 _C.DATA.NUM_WORKERS = 8
-
+# Number of data classes
+_C.DATA.NUM_CLASSES = 1000
```
Comment (ShawnXuan) [inline cv/classification/config.py:38]: 这两行是必须的吗？

File: cv/classification/main.py [hunk 2]
```diff
@@ -288,7 +288,7 @@ def train_one_epoch(
 
         one_sample_time.record(samples.size(0) * flow.env.get_world_size())
         loss_meter.record(loss.cpu().detach(), targets.size(0))
-        
+
```
Comment (ShawnXuan) [inline cv/classification/main.py]: 这一行可以不加

File: cv/classification/config.py (hunk 1 above)
Comment (akeeei) [reply cv/classification/config.py:38]: 因为有的数据集他们的类别数不同，这个可以传参调整最后输出层的神经元数量。我之前没加这两行会报错

**Review by ShawnXuan — APPROVED** [review]
